services/app-web/src/common-code/data/statePrograms.json

```diff
@@ -816,26 +822,20 @@
                 {
                     "id": "09dae8a2-df85-40e7-815e-0262fc49528c",
                     "fullName": "NJ FamilyCare MLTSS",
-                    "name": "FamilyCare MLTSS",
+                    "name": "MLTSS",
                     "isRateProgram": true
                 },
                 {
                     "id": "060cff75-f126-4248-95d1-846b3305a886",
                     "fullName": "NJ FamilyCare Acute",
-                    "name": "FamilyCare Acute",
+                    "name": "Acute",
                     "isRateProgram": true
                 },
                 {
                     "id": "da99a50b-accc-4849-b4a6-ce1436391dc2",
                     "fullName": "NJFamilyCare",
                     "name": "NJ FamilyCare",
                     "isRateProgram": false
-                },
-                {
-                    "id": "3e7133b4-f5bb-42a3-a076-f0c59beb9e10",
```

Comment:
NJ has submissions in prod, should we make sure this program was not used in their submission?